ups/cp_pipe.cfg

```diff
@@ -0,0 +1,26 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+# Dependencies that provide header files and or libraries should be included here.
+# Pure-Python dependencies do not need to be included.
+# Packages that use pybind11 or boost_tests should declare them as build dependencies.
+# Otherwise, the rules for which packages to list here are the same as those for
+# table files.
+dependencies = {
+    "required": ["utils", "ndarray"],
+    "buildRequired": ["boost_test", "pybind11", "ndarray", "pex_exceptions"],
```

Comment:
You need to revise these lists. Only include those that you use directly.

Reply:
Sorry about this, this is another instance of our template package misleading people.